client/css/bar.css

```diff
@@ -0,0 +1,60 @@
+.hero {
+  display: grid;
+  align-content: center;
+  justify-items: center;
+  width: 100%; /* don't use 100vw, that will cause horizontal scrollbar */
+  height: 100vh;
+}
+
+.svg {
+  width: 100%;
+  height: 100%;
+}
```

Comment:
ignore changes on bar.css and bar.js .... I don't know why they are appearing

Reply:
is this PR supposed to be based against `master`?

Reply:
I will git pull everything from master :'(